docs/_static/custom_font_example.css

```diff
@@ -0,0 +1,64 @@
+/* comic-neue-latin-300-normal */
+@font-face {
```

Comment:
Perhaps we can reduce the number of variants here, to avoid adding to the repo size unnecessarily?

Reply:
I wasn't sure which weights I needed, but it is a good idea... I'll look into it.

Reply:
I think if you skip variants the browser will just synthesize them, so the extra variants just increase the quality.  The default is 400.  But I have not verified to confirm.

Reply:
Default is 400, and it looks like rxjs might be doing some kind of synthesis with the font 🤷🏼‍♂️ .

I have removed all but the 400 weight for "Comic Neue" and "Comic Mono", and the output looks acceptable for an example. There's a slight decrease in quality but whatever.